sound/soc/sof/intel/hda-dsp.c

```diff
@@ -963,3 +963,65 @@ void hda_dsp_d0i3_work(struct work_struct *work)
 				    "error: failed to set DSP state %d substate %d\n",
 				    target_state.state, target_state.substate);
 }
+
+/*
+ * Some platforms do not support enabling/disabling secondary cores dynamically. Power up
+ * the requested core when used for the first time.
```

Comment:
did you mean to say that put() is not supported? that's not very clear from the comment above.

Reply:
I've added get/put for CNL?APL/ICL now but I dont think I can recommend using multi-core with dynamic pipelines on these platforms just yet.